osm-fudge/__main__.py

```diff
@@ -0,0 +1,27 @@
+import bk_tree
+import sys
+
+import levenshtein
+import jaccard
+
+distance_metrics = {
+	'levenshtein': levenshtein.find_levenshtein_distance,
+	'jaccard': jaccard.find_jaccard_similarity
+}
+
+with open(sys.argv[1], 'r') as file:
```

Comment:
Needs an explicit check if the argument is there; or argparse

Reply:
What you usually do is

```python
if __name__ == '__main__':
    main()
```

```python
def main():
    args = parse_arguments() ...
```

To mimic other language's main function and to have one clear entry into your program.